en/docs/WIP/create-and-publish-awslambda-api.md

```diff
@@ -0,0 +1,122 @@
+# Create and Publish an AWS Lambda API
+
+Follow the instructions in this tutorial to design and publish an AWS Lambda API.
+
+<html>
+   <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>For more information on AWS Lambda, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/welcome.html">What is AWS Lambda?</a></p>
+   </div> 
+</html>
+
+### Step 1 - Design a REST API
+
+1. Sign in to the API Publisher Portal.
+   
+    `https://<hostname>:9443/publisher` 
+   
+    Example: `https://localhost:9443/publisher`
+
+    Use your username and password to sign in.
+
+2. Click **CREATE API** and then click **Design a New REST API**.
+
+    [![Create a new REST API]({{base_path}}/assets/img/Learn/create-api-design-rest-api-link.png)]({{base_path}}/assets/img/Learn/create-a-rest-api.jpg)
+
+3. Enter API details and click **CREATE**.  
+
+    [![Create a test API]({{base_path}}/assets/img/Learn/create-test-api.png)]({{base_path}}/assets/img/Learn/create-test-api.png)
+
+    <html>
+      <div class="admonition note">
+         <p class="admonition-title">Note</p>
+         <p>You don't need to enter the Endpoint at this moment.</p>
+      </div> 
+    </html>
+
+Now, you have created a new REST API successfully. 
+
+### Step 2 - Add AWS Lambda endpoint
+
+1. Goto **Endpoints**
+2. Add **AWS Lambda** endpoint type
+
+    [![Select AWS Lambda endpoint]({{base_path}}/assets/img/Learn/Tutorials/endpoint-select-awslambda-endpoint.png)]({{base_path}}/assets/img/Learn/Tutorials/endpoint-select-awslambda-endpoint.png)
+
+3. Select **Access Method**
+
+    AWS SDK needs AWS credentials to invoke AWS Lambda functions. Access Method defines how you provide those AWS credentials. You can provide AWS credentials manually by selecting **Using stored AWS credentials** method. But if WSO2 API Manager is running on an Amazon EC2 instance, you can select **Using IAM role-supplied temporary AWS credentials** method.
+
+    <html>
```

Comment:
this note can also be a material theme note.

Reply:
changed